public/app/features/explore/state/reducers.ts

```diff
@@ -114,11 +115,16 @@ export const makeExploreItemState = (): ExploreItemState => ({
   update: makeInitialUpdateState(),
   latency: 0,
   supportedModes: [],
-  mode: null,
+  mode: (null as unknown) as ExploreMode,
```

Comment:
Shouldn't this be also `ExploreMode | null`? Same for `eventBridge`? Or is there some issue with that?

Reply:
Fixed. Just wasn't 100% sure with adding the null on those types as we don't set them to null anywhere else, so for the static type check to display an error if someone does that by mistake.

Reply:
@aocenas doubly checked - making those two variables possible to be nulls results in ~20 additional strict null errors. Not sure there's a point in fixing that now considering the `exploreMode` is going to be removed with the metrics & logs unification.